recipes/mget3/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "mget3" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mget3-{{ version }}.tar.gz
+  sha256: e722414b869f5c7775b9b4472db6a0ca42790ca0706b91b4551cdfa87c8b6621
+
+build:
+  # Our main reason for releasing mget3 on conda-forge is to support users of
+  # ArcGIS Pro 3.2.2 and later. ArcGIS Pro includes Anaconda Python 3.9 or
+  # later and only runs on Windows. So we're skipping Python < 3.9, osx, and
+  # linux. We depend on some packages that don't support Python 3.13 yet, so
+  # skipping Python >= 3.13.
+  skip: true  # [py<39 or py>=313 or py2k or osx or linux]
```

Comment:
It looks like this project should run on linux/osx (correct me if I'm wrong). I think it would be better to build it for all of the platforms it actually supports.

Reply:
It probably would work on osx. The problem is we don't have any Mac hardware or expertise so can't test it very well or respond to issues raised by osx users. If it is ok with conda-forge, we'd prefer not to offer osx builds until we have an osx maintainer signed up.

Reply:
It will run on Linux and we could enable that.

Reply:
That error message appears to be complaining about missing `g++`. I think you can get that by adding:

```
- {{ compiler('cxx') }}
```

to the `requirements`\`build` section

Reply:
Ok thanks! I added that and removed linux from the skip expression. Hopefully it will build now.

Reply:
Thanks, that worked! Hmm, now [linux_64 is failing further on](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1042813&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823), in `pip check`:
```
+ pip check
mget3 3.0.3 requires scikit-fmm, which is not installed.
WARNING: Tests failed for mget3-3.0.3-py311hfdbb021_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
```
This seems strange, because above that, it seems to say it is installing scikit-fmm:
```
The following NEW packages will be INSTALLED:

    ...
    scikit-fmm:         2024.5.29-py311hd037940_0     conda-forge
```
This could be related to a complex problem with scikit-fmm, explained by our comment in our `meta.yaml`:
```
  run:
    ...
    # After scikit-fmm 2023.4.2, the scikit-fmm team added support for Python
    # 3.12 but could not figure out how to build wheels yet with their new
    # tooling. Thus in our pyproject.toml, we have a constraint that installs
    # scikit-fmm 2023.4.2 on Python < 3.12, and skips it on Python >= 3.12
    # (the portions of our package that need it fail gracefully if it is not
    # importable). It does appear that conda-forge somehow contains Windows
    # builds of later versions of scikit-fmm, indicating that conda-forge is
    # capable of building these later versions. Therefore, we'd *like* to not
    # list a constraint for scikit-fmm in our meta.yaml recipe below, and thus
    # allow conda-forge users to gain the benefit of scikit-fmm on Python
    # 3.12. However, when testing our package, the conda-forge CI build script
    # somehow also applies the constraints in pyproject.toml, not just those
    # in the "run" section of meta.yaml here--maybe it invokes pip and pip
    # applies pyproject.toml? Anyway, when the conda-forge CI script tries to
    # install our mget3 package, after previously installing a scikit-fmm
    # version > 2023.4.2 (because we didn't constrain it in meta.yaml), the
    # script fails due to the pyproject.toml constraint on scikit-fmm
    # <=2023.4.2.
    #
    # To fix this, I'm adjusting the meta.yaml constraint on scikit-fmm to
    # match that of pyproject.toml, which is on Windows we install scikit-fmm
    # 2023.4.2 or earlier, but only if we have Python earlier than 3.12. For
    # non-Windows platforms, we do not constrain scikit-fmm's version.
    - scikit-fmm <=2023.4.2  # [win and py<312]
    - scikit-fmm  # [not win]
```
For reference, here is the relevant part of `pyproject.toml`:
```
dependencies = [
    ...
    "scikit-fmm; sys_platform!='win32'",                                       # Install the latest scikit-fmm on non-Windows platforms
    "scikit-fmm<=2023.4.2; sys_platform=='win32' and python_version<'3.12'",   # On Windows, wheels are only currently available scikit-fmm 2023.4.2 for Python 3.11 and earlier. For 3.12, just don't install scikit-fmm.
]
```
Assuming that conda did install `scikit-fmm` version `2024.5.29-py311hd037940_0`, why would `pip check` then fail with `mget3 3.0.3 requires scikit-fmm, which is not installed`. Any ideas?

Reply:
I just created an environment locally (osx) using `mamba create -n scikit-fmm python=3.12 scikit-fmm pip` and then did `pip list`, which gives back:

```
Package    Version
---------- -------
numpy      2.1.1
pip        24.2
setuptools 75.1.0
wheel      0.44.0
```

I'm wondering if something about its metadata is broken that `pip` isn't seeing it.

Reply:
`conda list` picks it up though:

```
...
scikit-fmm                2024.9.16       py312h98e817e_1    conda-forge
...
```

Reply:
@conda-forge/help-python any thoughts on why `pip check` isn't seeing `scikit-fmm` when it is in the environment?

Reply:
@synapticarbors, how long does it typically take for them to respond, and will this even come up on their radar if some of the checks are failing, when there are (as of this moment) 600 PRs outstanding? Our target community for the conda-forge packaging is Windows. If its going to be hard to find the support to get Linux working, we'd rather just go with Windows for now. Rather than adding to @conda-forge/help-python's workload right now, I could just remove Linux again. What do you think? (And apologies for hassling you and them.)

Reply:
Sorry you haven't gotten any additional feedback. Let's just back out the linux build and then fix it at a later date in the feedstock.

Reply:
Thanks for being flexible about this. The checks have all passed. Is there anything else we need to do for this PR? If not, what should happen now?